tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "module": "amd",
     "moduleResolution": "node",
-    "target": "es5",
+    "target": "es6",

```

Comment:
That was my best guess as well. For compatibility reasons, I would want to keep it at ES5 since this change would isolate anyone who is not running windows 10 and uses IE. Users in many enterprise environments for example.